loopchain/blockchain/blocks/v0_3/block.py

```diff
@@ -25,6 +27,26 @@ class BlockHeader(BaseBlockHeader):
     def complained(self):
         return self.leader_votes_hash != Hash32.empty()
 
+    @property
+    def prep_changed(self) -> Optional[PrepChangedReason]:
```

Comment:
I think its naming could raise confusion to return bool type (prep changed or not).
How about `prep_changed_reason` or...

as normal method: get_prep_changed_reason? I saw in your code below:
```python
            if self._block_manager.epoch.leader_id != ChannelProperty().peer_id \
                    and not candidate_block.header.prep_changed:
```
It could be:
```python
            if self._block_manager.epoch.leader_id != ChannelProperty().peer_id \
                    and not candidate_block.header.get_prep_changed_reason():
```

Reply:
It will develop further to feature 'keep order'. In this PR it was used as a kind of boolean (of course It's not actual bool value).